source/Projectile.js

```diff
@@ -32,7 +30,7 @@ export default class Projectile extends Pixi.Sprite {
   onCollision(collidedWith) {
       game.removeChild(this)
       this.destroy()
-      console.log("hit")
+      // console.log("hit")
```

Comment:
Removed.

There has never been a more satisfying GIF of disdain in the history of the Internet, btw.